winlogbeat/beater/winlogbeat.go

```diff
@@ -110,7 +110,7 @@ func (eb *Winlogbeat) setup(b *beat.Beat) error {
 	config := &eb.config
 
 	var err error
-	eb.checkpoint, err = checkpoint.NewCheckpoint(config.RegistryFile, 10, 5*time.Second)
+	eb.checkpoint, err = checkpoint.NewCheckpoint(config.RegistryFile, 10, config.RegistryFlush)
```

Comment:
In order to make increasing the value more effective at throttling the number of writes, can you disable the max number of updates parameter (like change the `10` to a `-1` and update the checkpoint package to handle that). Or completely remove that parameter from checkpoint.